contexts/compound-and-earn-context.js

```diff
@@ -339,6 +339,7 @@ export function CompoundAndEarnProvider({ children }) {
 
     setIsTransacting({ deposit: true });
     try {
+      var gaugeAmount = ethers.BigNumber.from("0");
```

Comment:
if `gaugeAmount` is only used within `try` consider using `let` to keep its scope limited to the `try` block